app/components/termsoverlay/termsoverlay.js

```diff
@@ -13,82 +13,217 @@
  * You should have received a copy of the License along with this program; if
  * not, you can obtain one from Tidepool Project at tidepool.org.
  */
-
 var React = require('react');
+var LoginNav = require('../loginnav');
 
 var TermsOverlay = React.createClass({
   propTypes: {
     onSubmit: React.PropTypes.func,
-    trackMetric: React.PropTypes.func.isRequired
+    trackMetric: React.PropTypes.func.isRequired,
+    ages: React.PropTypes.object,
+    messages: React.PropTypes.object
+  },
+  getDefaultProps: function(){
+    return {
+      ages: {
+        OF_AGE : { value: '>=18', label: ' I am 18 years old or older.'},
+        WITH_CONSENT : { value: '13-17', label: ' I am between 13 and 17 years old. You\'ll need to have a parent or guardian agree to the terms on the next screen.' },
```

Comment:
its the same as the original language proposed by @HowardLook can change if needed